db/migrate/20241124050607_create_tickets_edit_person_fields.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateTicketsEditPersonFields < ActiveRecord::Migration[7.2]
+  def change
+    create_table :tickets_edit_person_fields do |t|
+      t.bigint :tickets_edit_person_id, null: false
+      t.string :field_name, null: false
+      t.string :old_value, null: false
+      t.string :new_value, null: false
```

Comment:
Shouldn't this be JSON?

Reply:
Actually string is also working fine because the request from frontend is anyway coming in string format. Currently there is no use-case to convert it to date or any other type. But it may come in future for this same ticket type. Even then, having this as string will be sufficient.

Reply:
Then at least change it to `t.text`, because the length of a `string` field in the DB is quite limited.